### PR TITLE
added support for Linode

### PR DESCRIPTION
**Pull request by tomschlick**

### CODE REVIEW

File: src/Linode/Provider.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    /**
+     * {@inheritdoc}
+     */
+    protected function mapUserToObject(array $user)
+    {
+        return (new User())->setRaw($user)->map([
+            'id'     => $user['uid'],
+            'name'   => $user['username'],
+            'email'  => $user['email'],
+            'avatar' => 'https://www.gravatar.com/avatar/'.md5($user['email']),
```
Comment (atymic) [inline src/Linode/Provider.php:61]: do we need to return this? I think it's prob best to just return null and someone can make a gravatar url if they want?

**Review by atymic — COMMENTED** [review]
Just one minor thing

File: src/Linode/Provider.php (hunk 1 above)
Comment (tomschlick) [reply src/Linode/Provider.php:61]: I’m fine with omitting it. It was copied from the DigitalOcean configuration.

Comment (atymic) [reply src/Linode/Provider.php:61]: Thanks :)

**Review by atymic — CHANGES_REQUESTED** [review]
Could you please add it to the monorepobuilder.yml split script.

**Review by atymic — APPROVED** [review]

**Pull request merged**